packages/vm/src/instance.rs

```diff
@@ -237,6 +237,27 @@ where
             Function::new_typed_with_env(&mut store, &fe, do_db_next),
         );
 
+        // Get next key of iterator with ID `iterator_id`.
+        // Creates a region containing the key and returns its address.
+        // Ownership of the result region is transferred to the contract.
+        // An empty key means no more elements.
```

Comment:
We should be able to encode the None case as a 0 pointer in the return value (same as "db_read") both here and for `db_next_value`. 

We may have made a bit of a mistake and not do that for `db_next`. But at least for the new imports this can save us new custom encodings.

Reply:
Oh, that makes a lot more sense.